### PR TITLE
Update remote-state-data.mdx

### PR DESCRIPTION
**Pull request by achuchulev**
In order to be able to use the outputs from a workspace with the `tfe_outputs` data source, you can choose currently one of the following options:

1) Configure the "Remote state sharing" option in the "General Settings" of the source workspace. (TF Cloud relies on the short-lived Terraform Cloud API token used during the run)
2) Provide the "tfe" provider with an API token that has the "Read state outputs" permission for the source workspace.

The "important" section within documentation [here](https://github.com/hashicorp/terraform/edit/main/website/docs/language/state/remote-state-data.mdx#the-terraform_remote_state-data-source) states the opposite but this is required to use `tfe_outputs` in case of 1).

### CODE REVIEW

File: website/docs/language/state/remote-state-data.mdx
Comment (laurapacilio) [inline website/docs/language/state/remote-state-data.mdx:17]: Hi! Thank you for submitting this! I'm thinking through the best way to handle. Specifically, this note is quite long and I'm wondering if we can fix the problem you identified without making it longer. 

I'm wondering if the problematic statement is "it does not require full access". I wonder if instead, we could change this to:

"The `tfe_outputs` data source is more secure because it does not require write access to workspace state to fetch outputs"

This way, we're clarifying that it does require some access, but we're keeping the sentence a bit more concise. Let me know what you think!

**Review by laurapacilio — COMMENTED** [review]
I have a thought on this. Specifically, I don't want to go into too much detail in this note. Notes are meant to be very short asides to the user, but I do want to fix the problematic language you identified. Let me know what you think of my suggestion! 

I appreciate you opening this PR and bringing this to our attention!

File: website/docs/language/state/remote-state-data.mdx
Comment (achuchulev) [reply website/docs/language/state/remote-state-data.mdx:17]: Hi @laurapacilio,

I agree that the proposed change makes the sentence more concise and it looks good to me so let's have the documentation updated that way.

Thanks for the help here!

Comment (laurapacilio) [inline website/docs/language/state/remote-state-data.mdx:17]:
```suggestion
~> **Important:** We recommend using the [`tfe_outputs` data source](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/data-sources/outputs) in the [Terraform Cloud/Enterprise Provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs) to access remote state outputs in Terraform Cloud or Terraform Enterprise. The tfe_outputs data source is more secure because it does not require write access to workspace state to fetch outputs.
```

**Review by laurapacilio — CHANGES_REQUESTED** [review]
Excellent! I am leaving you the correction we discussed :-) Once you merge that into your PR, we can get this changed merged into the docs. Thank you!

File: website/docs/language/state/remote-state-data.mdx
Comment (achuchulev) [reply website/docs/language/state/remote-state-data.mdx:17]: It seems that I am lacking write access to the repo so here is the PR [#31514](https://github.com/hashicorp/terraform/pull/31514) to update the "update-important-section" branch. Thanks!

**Comment by laurapacilio** [comment]
@achuchulev - should I close this one then? Let me know how I can help - I can also merge things in for you if you need me to!

**Comment by achuchulev** [comment]
@laurapacilio I will close this one in favor of PR #31514 so merge the latter for me, please. Thanks!

**Pull request closed without merging**
